src/components/KeyboardInteraction/index.ts

```diff
@@ -32,12 +32,16 @@ class KeyboardInteractionInner extends Component<Props> {
     switch (e.code) {
       case 'ArrowLeft':
       case 'KeyH':
-        this.props.previousSong();
+        if (!e.altKey){
```

Comment:
So, a couple things I should have made more clear.

1. Actions should be be ignored if any modifier key is pressed. There are four modifier keys, Alt, Shift, Ctrl and Meta.

2. All actions should be ignored if any modifier key is pressed. Currently modifiers are ignored for keys associated with the previous and next song actions. They should be ignored for keys associated with all actions.

3. preventDefafult is run even when `e.altKey` is true (line 55). This means that the browser will not be able to handle the event because the page has already marked it as handled.